src/modules/price-router/PriceRouter.sol

```diff
@@ -8,9 +8,13 @@ import { IChainlinkAggregator } from "src/interfaces/external/IChainlinkAggregat
 import { SafeCast } from "@openzeppelin/contracts/utils/math/SafeCast.sol";
 import { Math } from "src/utils/Math.sol";
 import { Address } from "@openzeppelin/contracts/utils/Address.sol";
-import { ICurvePool } from "src/interfaces/external/ICurvePool.sol";
-import { IAaveToken } from "src/interfaces/external/IAaveToken.sol";
+import { Extension } from "src/modules/price-router/Extensions/Extension.sol";
 
+import { UniswapV3Pool } from "src/interfaces/external/UniswapV3Pool.sol";
+import { OracleLibrary } from "@uniswapV3P/libraries/OracleLibrary.sol";
+
+// TODO so we could have some external watch dog contract that can freeze assets if there are issues detected
```

Comment:
Not at the moment, we could in the future but doing so will increase gas costs further. 

It is very important for lending protocols to have backup oracles so that liquidations can always happen, but in our case it isn't nearly as important. Yes if some assets chainlink feed were reverting, the multisig could brick cellars trying to price that asset by doing nothing, but if that happens, governance has the ability to force-ably remove the old multisig and replace it with one that will actually fix the problem.

Reply:
Discussed and resolved, bad sources will result in Cellars being paused essentially while new sources are linked to.